plugins/modules/cloud/scaleway/scaleway_compute_private_network.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway VPC management module
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_compute_private_network
+short_description: Scaleway compute - private network management
+version_added: 5.1.0
+author: Pascal MANGIN (@pastral)
+description:
+    - This module add or remove a private network to a compute instance
+      (U(https://developer.scaleway.com)).
+extends_documentation_fragment:
+- community.general.scaleway
+
+
+options:
+  state:
+    type: str
+    description:
+     - Indicate desired state of the VPC.
+    default: present
+    choices:
+      - present
+      - absent
+
+  project:
+    type: str
+    description:
+      - Project identifier.
+    required: true
+
+  region:
+    type: str
+    description:
+     - Scaleway region to use (for example C(par1)).
+    required: true
+    choices:
+      - ams1
+      - EMEA-NL-EVS
+      - par1
+      - EMEA-FR-PAR1
+      - par2
+      - EMEA-FR-PAR2
+      - waw1
+      - EMEA-PL-WAW1
+
+  compute_id:
+    type: str
+    description:
+    - ID of the compute instance (see M(community.general.scaleway_compute)).
+    required: true
+
+  private_network_id:
+    type: str
+    description:
+    - ID of the private network (see M(community.general.scaleway_private_network)).
+    required: true
+
+'''
+
+EXAMPLES = '''
+- name:
+  community.general.scaleway_compute_private_network:
+    project: '{{ scw_project }}'
+    state: present
+    region: par1
+    compute_id: "12345678-f1e6-40ec-83e5-12345d67ed89"
+    private_network_id: "22345678-f1e6-40ec-83e5-12345d67ed89"
+  register: nicsvpc_creation_task
+
+- name:
+  community.general.scaleway_compute_private_network:
+    project: '{{ scw_project }}'
+    state: absent
+    region: par1
+    compute_id: "12345678-f1e6-40ec-83e5-12345d67ed89"
+    private_network_id: "22345678-f1e6-40ec-83e5-12345d67ed89"
+
+'''
+
+RETURN = '''
+scaleway_compute_private_network:
```

Comment:
In the code below, the return value is called `scaleway_private_network`:
```suggestion
scaleway_private_network:
```

Reply:
I change the name in the code below  because scaleway_private_network is misleading with the module scaleway_private_network:
scaleway_private_network -> module gfor virtual network
scaleway_compute -> module for VM
scaleway_compute_private_network -> module for NIC to link VM and VPc ( a little like vmware_guest_network module )

Sorry. Thanks for the review :)